src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.td

```diff
@@ -54,7 +54,7 @@ class VariadicSizeIsGT<int N> : Constraint<
 >;
 
 // Create a unit constant that will be used as none input.
-def CreateUnitConstant : NativeCodeCall<"$_builder.create<ONNXNoneOp>($_loc)">;
+def CreateNoneConstant : NativeCodeCall<"$_builder.create<ONNXNoneOp>($_loc)">;
```

Comment:
It makes a lot of sense. How about using `CreateNoneValue` instead of `CreateNoneConstant`? as you commented its does not matter whether it's a constant or not. Furthermore, it would be consistent with the check function `isNoneValue`.

Reply:
good idea

done